src/main/java/by/shop/App.java

```diff
@@ -8,6 +10,11 @@
 {
     public static void main( String[] args )
     {
-        System.out.println( "Hello World!" );
+        Client klient1 = new Client();
+        Celler celler1 = new Celler();
```

Comment:
Lesh, correct the name, please to Seller. The same with `klient` - should be `client`
And why do you use 1 in variables. Do you have second seller and client?

Reply:
Eliminated